src/encoded/schemas/output_file.json

```diff
@@ -93,6 +93,14 @@
         },
         "read_pair_number": {
             "$merge": "encoded:schemas/unaligned_reads.json#/properties/read_pair_number"
+        },
+        "role": {
```

Comment:
Should be added to MWF `custom_pf_fields`: https://github.com/smaht-dac/smaht-portal/blob/main/src/encoded/schemas/meta_workflow.json#L120-L143.